caramel/scripts/tool.py

```diff
@@ -174,7 +180,7 @@ def csr_resign(ca, lifetime_short, lifetime_long, backdate):
     with concurrent.futures.ThreadPoolExecutor(max_workers=16) as executor:
         try:
             csrlist = models.CSR.refreshable()
-        except:
+        except Exception:
```

Comment:
"No CSR's found" almost certainly isn't accurate for every kind of `Exception` instances.